mugoh/qstioner
# #163846336 Edit Created User Resources

#### What does this PR do?
Allow users to make changes to any resource they've posted to the application

#### Description of Task to be completed?
- A user can edit a resource
- A user delete a resource
 - A resource in this case can be:
1) Meetup
2) Comments
3) Questions

#### How should this be manually tested?
Ensure the server is running. Follow the steps given [here](https://github.com/hogum/qstioner-v2) to do so.
1. Clone the repo in your machine
``` shell
$ git clone https://github.com/hogum/qstioner
```
2. Navigate to the directory `qstioner` where you have the repo cloned
``` shell
$ cd qstioner
```
3. Checkout this Feature branch
``` shell
$ git checkout ft-edit-resources-163846336
```
4. Open the `index.html` file
``` shell
$ browser-name index.html
```
or
Simply  load up on your preferred browser
5. Navigate to the sign in page and sign in with account credentials.
- email: `admincow@mammals.milk` password: `pa555word`

6. Create a meetup on the admin dashboard.

7. Open  this meetup edit page by clicking on it's edit button.
The meetup will be displayed with an edit and a delete Button. 

8. Click on the edit button. A form to enter new meetup details should show up. Make an edit to any present meetup detail an submit.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#163846336](https://www.pivotaltracker.com/story/show/163846336)
